src/curated-corpus/helpers/helperFunctions.ts

```diff
@@ -175,3 +181,51 @@ export const getLastScheduledDayDiff = (
     new Date(mostRecentScheduleDate).getTime();
   return Math.abs(Math.ceil(daysDifference / (1000 * 3600 * 24)));
 };
+
+/**
+ * Formats an excerpt using rules based on the excerpt's language.
+ * @param language the language the passed excerpt is in
+ * @param excerpt excerpt to format
+ * @returns formatted excerpt
+ */
+export const applyExcerptFormattingByLanguage = (
+  language: CorpusLanguage,
+  excerpt: string,
+): string => {
+  // if excerpt is English, apply quotes formatting for EN
+  if (language === CorpusLanguage.En) {
+    applyCurlyQuotes(excerpt);
```

Comment:
missing a `return` here! (but see below for more refactoring.)

also - this function still works as expected for EN, even with this missing `return`. but why? 😉

Reply:
oo good catch! its always switching to the `else` block as nothing is explicitly returned in the EN `if` block. & they both return the same result